exe/manageiq-cross_repo

```diff
@@ -31,10 +31,15 @@ opts = Optimist.options do
     Can also be passed as a REPOS environment variable.
   EOS
 
+  opt :test_suite, <<~EOS, :type => :string, :default => ENV["TEST_SUITE"].presence || ""
+    Optional, the name of a rake test suite to pass to `bundle exec rake` e.g. spec:javascript.
```

Comment:
nit: Ideally, document what this does with a custom --script-cmd.

AFAICT, this doesn't actually pass it as arg `bundle exec rake ...`, it sets env var TEST_SUITE, which the manageiq repos use in the default command, right?

Reply:
:+1: this comment is actually out of date now thank you.  Instead of passing it to `bundle exec rake #{test_suite}` it is now passed as an env var which is typically used by the `.travis.yml`'s `script` directive.  Will update